src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/BlobContainerClientProvider.java

```diff
@@ -30,6 +31,12 @@ public BlobContainerClient get(TargetStorageAccount targetStorageAccount, String
             case BULKSCAN:
                 // retrieving a SAS token every time we're getting a client, but this will be cached in the future
                 return new BlobContainerClientBuilder()
+                    .configuration(Configuration
+                        .getGlobalConfiguration()
+                        // testing
+                        .put(Configuration.PROPERTY_HTTP_PROXY, "proxyout.reform.hmcts.net:8080")
+                        .put(Configuration.PROPERTY_HTTPS_PROXY, "proxyout.reform.hmcts.net:8080")
+                    )
```

Comment:
this will apply proxy to even main job running on aks . I would imagine you need it only for functional tests ?

Reply:
testing out ;). haven't marked PR as ready anyway but good point @nitinprabhu